mongomock/collection.py

```diff
@@ -36,6 +38,14 @@ class ReturnDocument(object):
 
     from mongomock.read_preferences import PRIMARY as _READ_PREFERENCE_PRIMARY
 
+    class Collation(object):
+        pass
+
+    def validate_collation_or_none(
+        value: Mapping[str, Any] | Collation | None
```

Comment:
We are still supporting Python versions below 3.10 so that the union operator can not be used but rather `typing.Union` has to be employed.